pytorch_toolkit/ote/ote/modules/registry.py

```diff
@@ -14,7 +14,7 @@
  limitations under the License.
 """
 
-from mmcv.utils import Registry
+from mmcv.utils.registry import Registry
```

Comment:
useless change, is not it?

Reply:
If `torch` is not installed the `mmcv` will not contain `Registry`. So, it really needed here.

Reply:
Yes, it is really funny, but it is so. (See `mmcv.utils.__init__.py` for details)
We should either install `torch` even in the environment with ote tests, or import it as `mmcv.utils.registry`.